src/templates/tpl-submissions.html

```diff
@@ -300,6 +300,10 @@
 				<input type="checkbox" id="notifyUser" class="afch-input" checked/>
 				<label for="notifyUser" class="afch-label">Notify submitter</label>
 			</div>
+			<div class="afch-option">
+                               <input type="checkbox" id="sendWelcome" class="afch-input" checked/>
```

Comment:
Per https://en.wikipedia.org/wiki/Wikipedia_talk:WikiProject_Articles_for_creation#Welcome_template, can we please make this unticked by default?